src/clappr-dash-shaka-playback.js

```diff
@@ -80,6 +80,14 @@ class DashShakaPlayback extends HTML5Video {
     return this._duration
   }
 
+  getFrameRate() {
+    if (this.videoTracks && this._currentLevelId) {
+      const track = this.videoTracks.filter((t) => t.id === this._currentLevelId)[0]
```

Comment:
Using `find` will return the first found item here. I think is the best for this scenario.

`const track = this.videoTracks.find((t) => t.id === this._currentLevelId)`